src/main/resources/alma/common/fields.xml

```diff
@@ -365,11 +365,16 @@
     </data>
   </entity>
 	-->
-  <call-macro name="regex-del-punctuation-end" dsource="245??.a" dname="title" />
-  <data name="title" source="245??.p">
+  <call-macro name="regex-del-punctuation-end" dsource="245??.a" dname="@title" />
+  <data name="@title" source="245??.p">
     <occurrence only="1" />
     <replace pattern="\s?\.?\s?$" with="" />
   </data>
```

Comment:
Not sure why introduce a variable here. You could also simply add the new "replaces" here.

Reply:
There are two sources of the title. I wanted to insert the use the replacement only once. 245.p is used if 245 .a does not exist.

Reply:
Ah, yeah I see.So leave that as it is.